.github/workflows/ci-build-binaries.yml

```diff
@@ -31,6 +31,7 @@ jobs:
     needs: generate-matrix
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
please add a step to the job that checks for one combination of `${{ matrix.os }}-${{ matrix.arch }}` and fails, so that we can see if all other jobs will continue. In fact I would make that change first without `fail-fast` to validate that other jobs will get cancelled if that one fails, and then apply the fix where only one of them will fail.

Reply:
@yurishkuro I already did some testing and documented it in the PR description and the commits are available in the history. Are you looking for something different?